pkg/views/workspace/selection/view.go

```diff
@@ -161,5 +189,31 @@ func (d ItemDelegate[T]) Spacing() int {
 }
 
 func (d ItemDelegate[T]) Update(msg tea.Msg, m *list.Model) tea.Cmd {
+	i, ok := m.SelectedItem().(item[T])
+	if !ok {
+		return nil
+	}
+
+	m.StatusMessageLifetime = time.Millisecond * 2000
+
+	var title string
+	switch msg := msg.(type) {
+	case tea.KeyMsg:
+		switch {
```

Comment:
For the code style to be in sync with our other models, please update this like this example:
```golang
	case tea.KeyMsg:
		switch keypress := msg.String(); keypress {
		case "ctrl+c":
```